### ISSUE TITLE
Tree component, integrated with Vuex, doesn't preserve functions in data property

### ISSUE THREAD

**Issue by become-iron** [amsik/liquor-tree#138]
Demo: https://jsfiddle.net/9zLhndy6/20/

Way to reproduce:
1. Click button to log out tree state -- function is present
2. Change tree state by click on checkbox
3. Click button to log out state -- function disappeared

**Comment by amsik** [comment]
Hi.

It's correct behaviour. Tree is using `JSON.stringify` method. That's why functions disappears after change model.

A model is data only. Do not store there anything other than simple data. Use events instead

**Comment by become-iron** [comment]
@amsik 
It took me a while to find out. Please, add the corresponding note in documentation

**Comment by amsik** [comment]
Please check MDN: https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify

**Comment by become-iron** [comment]
Of course, I know how JSON.stringify does work. The issue is it wasn't clear that JSON.stringify is used under hood

**Comment by amsik** [comment]
Agree with you (about JSON.stringify is used under hood). But I can't understand why you store function for tree item?

**Comment by become-iron** [comment]
I used function in config because it was convenient for my task. And I don't do it now

Could you update docs on this topic?
